src/txTransformers.ts

```diff
@@ -84,14 +88,15 @@ export const transformTxBody = (txBody: TransactionBody): TransactionBody => ({
     requiredSigners: transformOptionalList(txBody.requiredSigners),
     collateralReturnOutput: txBody.collateralReturnOutput && transformTxOutput(txBody.collateralReturnOutput),
     referenceInputs: transformOptionalList(txBody.referenceInputs),
+    metadataHash: transformAuxiliaryDataHash(auxiliaryData),
```

Comment:
Shouldn't we rename `metadataHash` to `auxiliaryDataHash`? That's how it is now called in CDDL. And for people not familiar with the old history it might be confusing.

Reply:
I thought about that too, but I would rather not create breaking changes 🤔 .. I tried deprecating `metadataHash` and adding `auxiliaryDataHash` as a new, duplicit field. This would complicate `serializeTxBody` since we would have to check that only one of the two is defined and we should include that in the transactions. This would cause errors on clients anyways, so I'm not sure 🤔 Maybe it's not such a big deal to just rename the field. Perhaps we could then release this fix separately and create another release with the rename so clients are not forced to update at least 🤷 

What do you think?

Reply:
Maybe there aren't even any clients outside of our reach.

Reply:
Discussed offline: we'll rename it.